### PR TITLE
Allow Pip install of EPLaunch

### PR DESCRIPTION
**Pull request by Myoldmopar**
Minor tweaks to allow Pip installing from the repo.  Not intending on posting to PyPi, at least not anytime soon.  This change will enable some Python packaging/testing/prototyping going on ahead of 22.2 release.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1 @@
+include eplaunch/interface/main_icon.ico
```
Comment (Myoldmopar) [inline MANIFEST.in:1]: When building a Python wheel, you need to declare non-python-related files that should be included.  This manifest does just that.

File: eplaunch/__init__.py [hunk 2]
```diff
@@ -3,5 +3,5 @@
 # If you waste time on it, please add it here:
 # Total wasted time: 12 minutes
 NAME = "EP-Launch"
-VERSION = "3.0.0"
+VERSION = "3.0.1"
```
Comment (Myoldmopar) [inline eplaunch/__init__.py:6]: Bumped this just to see that it was getting the correct version during testing.  Open to discussion on version number.

File: eplaunch/runner.py [hunk 3]
```diff
@@ -2,6 +2,12 @@
...
-app.MainLoop()
+
+def main():
+    gettext.install("app")  # replace with the appropriate catalog name
+    app = EpLaunchApplication(0)
+    app.MainLoop()
+
+
+if __name__ == "__main__":
+    main()
```
Comment (Myoldmopar) [inline eplaunch/runner.py:13]: Added a `main()` function which can be added to the Python packaging `entry_points` keyword.  Then added a call to this main function inside here so it should result in no change in behavior.

File: setup.py [hunk 4]
```diff
@@ -1,11 +1,31 @@
...
+    long_description_content_type='text/markdown',
+    test_suite='nose.collector',
+    tests_require=['nose'],
+    keywords='energyplus',
+    include_package_data=True,
+    install_requires=['wxpython', 'pypubsub'],
+    entry_points={
+        'console_scripts': ['eplaunch3=eplaunch.runner:main'],
+    },
+    python_requires='>=3.5',
```
Comment (Myoldmopar) [inline setup.py:30]: Added the required pieces to get EPLaunch building a wheel and installing.  Seems to work ok for me.

**Review by Myoldmopar — COMMENTED** [review]
I already saw that CI was having an issue, so I'll check that out.  Otherwise these changes should be harmless and just enable pip installing.

**Comment by JasonGlazer** [comment]
@Myoldmopar thanks for doing this. Good step toward getting the installation to be easier.

**Comment by jmarrec** [comment]
I get an error here:

```
$ pip install .
Processing /home/julien/Software/Others/EnergyPlusApps/EP-Launch
  Preparing metadata (setup.py) ... done
Collecting wxpython
  Using cached wxPython-4.2.0.tar.gz (71.0 MB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [8 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-blhw8tk2/wxpython_fc994fdfa60b42e1815b5974cb64955a/setup.py", line 27, in <module>
          from buildtools.config import Config, msg, opj, runcmd, canGetSOName, getSOName
        File "/tmp/pip-install-blhw8tk2/wxpython_fc994fdfa60b42e1815b5974cb64955a/buildtools/config.py", line 30, in <module>
          from attrdict import AttrDict
      ModuleNotFoundError: No module named 'attrdict'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

```

**Comment by jmarrec** [comment]
https://github.com/wxWidgets/Phoenix/issues/2225

**Comment by Myoldmopar** [comment]
This is now wildly out of date and not necessary, converting the GUI to Tk makes all of this nonsense unnecessary, and it is already being hosted on PyPi now from another dev branch which should be merged in the near term.

**Pull request closed without merging**